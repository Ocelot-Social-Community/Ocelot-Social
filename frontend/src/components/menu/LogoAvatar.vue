<template>
  <v-avatar color="#333" class="ma-2 pa-1" :image="Logo" size="48" />
</template>

<script lang="ts" setup>
import Logo from '#assets/it4c-logo2-clean-bg_alpha-128x128.png'
</script>
