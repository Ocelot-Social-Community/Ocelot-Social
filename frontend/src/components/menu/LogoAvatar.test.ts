import { mount } from '@vue/test-utils'
import { beforeEach, expect, describe, it } from 'vitest'

import LogoAvatar from './LogoAvatar.vue'

describe('LogoAvatar', () => {
  const Wrapper = () => {
    return mount(LogoAvatar)
  }
  let wrapper: ReturnType<typeof Wrapper>

  beforeEach(() => {
    wrapper = Wrapper()
  })

  it('renders', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
