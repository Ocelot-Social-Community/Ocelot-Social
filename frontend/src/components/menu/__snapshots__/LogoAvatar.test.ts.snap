// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`LogoAvatar > renders 1`] = `
<div
  class="v-avatar v-theme--light v-avatar--density-default v-avatar--variant-flat ma-2 pa-1"
  style="background-color: #333; color: #fff; caret-color: #fff; width: 48px; height: 48px;"
>
  <div
    aria-label=""
    class="v-responsive v-img v-img--booting"
  >
    <div
      class="v-responsive__sizer"
    />
    
    
    <transition-stub
      appear="true"
      css="true"
      name="fade-transition"
      persisted="false"
    >
      <img
        alt=""
        class="v-img__img v-img__img--cover"
        src="/src/assets/it4c-logo2-clean-bg_alpha-128x128.png"
        style="display: none;"
      />
    </transition-stub>
    <transition-stub
      appear="false"
      css="true"
      name="fade-transition"
      persisted="false"
    >
      <!---->
    </transition-stub>
    <!---->
    <!---->
    <!---->
    
    
    <!---->
  </div>
  
  <!---->
  <span
    class="v-avatar__underlay"
  />
  
</div>
`;
